tests/test_db_setup.py

```diff
@@ -456,6 +456,8 @@ class TestNativeMigrations:
     """ Tests for Django Migrations """
 
     def test_no_migrations(self, django_testdir) -> None:
+        skip_if_sqlite_in_memory()
```

Comment:
Why?

Reply:
When the code actually works, migrations are disabled. In order for the test to actually load, we need to first run migrations, and then run the tests with `--nomigrations`. But there is no way to save the migrated state of the db using `sqlite_in_memory` the db is not maintained from one run to the next.